lib/travis/build/script/node_js.rb

```diff
@@ -47,15 +44,47 @@ def cache_slug
 
         private
 
+          def convert_legacy_nodejs_config
+            # TODO deprecate :nodejs
+            # some old projects use language: nodejs. MK.
+            config[:node_js] = config[:nodejs]
+          end
+
+          def node_js?
+            !!config[:node_js]
+          end
+
           def version
             @version ||= begin
-              # TODO deprecate :nodejs
-              version = config[:node_js] || config[:nodejs] # some old projects use language: nodejs. MK.
+              version = config[:node_js] || 'stable'
```

Comment:
I would prefer not to default to `stable` just yet. We need to discuss any implications this may have with the team.